src/index.ts

```diff
@@ -70,6 +70,15 @@ const rules = {
       },
     },
   ],
+  '@typescript-eslint/class-methods-use-this': [
```

Comment:
did you forget to set `"class-methods-use-this": "off",`? 
see <https://typescript-eslint.io/rules/class-methods-use-this/>
> ```js
> module.exports = {
>   "rules": {
>     // Note: you must disable the base rule as it can report incorrect errors
>     "class-methods-use-this": "off",
>     "@typescript-eslint/class-methods-use-this": "error"
>   }
> };
> ```

Reply:
Hrm, I thought there's test coverage for that. Sorry. Thank you.

Reply:
I remember now. I noticed that it seems that this implicitly occurs. I figured that [this line](https://github.com/mightyiam/eslint-config-love/pull/1644/files#diff-447484306515933c35c0576e4c250013bcf55ddac097d54d2167bcaaf389db94R109) proves that. Would you agree?

Reply:
https://discord.com/channels/1026804805894672454/1262278361183420467/1262278361183420467

Reply:
> I remember now. I noticed that it seems that this implicitly occurs. I figured that [this line](https://github.com/mightyiam/eslint-config-love/pull/1644/files#diff-447484306515933c35c0576e4c250013bcf55ddac097d54d2167bcaaf389db94R109) proves that. Would you agree?

not at all.

Reply:
I looked into it some more and even [rewrote the test to be more friendly](https://github.com/mightyiam/eslint-config-love/pull/1656). I see no problem, sorry.

Reply:
Here is where turning base rules off happens:
https://github.com/mightyiam/eslint-config-love/blob/32e863a510dfeedee924d4b45344a711f8eb0676/src/index.ts#L398-L403